contents/tutorials/hogql-date-time-filters.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using HogQL for advanced time and date filters
+date: 2023-05-19
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-9.png
+tags: ["hogql", "insights"]
+---
+
+Because there are infinite ways to break down time, there are infinite ways to filter based on time. HogQL unlocks more of these in PostHog, and in this tutorial, we go through examples of how to use do that.
+
+To add a HogQL filter, create a new insight, open the filter dropdown, click "Add filter" below your data series, select HogQL from the options, write your expression, and click "Add HogQL expression." HogQL filters are available on every type of insight from trends to funnels to lifecycle.
+
+![hogql.mp4](../images/tutorials/hogql-date-time-filters/hogql.mp4)
+
+## Accessing your data’s dates and times
+
+To filter on dates and times, you need access to those values. You can find a full list of events, properties, and types in your [data management tab](https://app.posthog.com/data-management/events), but here is a summary of the important points:
+
+- You can access the time events happened with the `timestamp` property which is automatically set on events.
+- Persons have a `created_at` property which you can access with `person.created_at`.
+- For custom event or person properties, such as `subscribed_at`, you can access them with `person.properties.subscribed_at`.
+
+> **Note:** you might need to convert strings to date time objects with the `toDateTime()` function or convert date time objects to strings with `toString()`.
+
+The [ClickHouse SQL statements](https://clickhouse.com/docs/en/sql-reference) we built HogQL on also have useful helper functions to know when it comes to using dates. These include:
+
+- `now()`: the current date and time at the moment of query analysis.
+- `today()`: the current date at the moment of query analysis.
+- `yesterday()`: yesterday’s date at the moment of query analysis.
```

Comment:
should we include a link to a more comprehensive list of helper functions related to date+time?

Reply:
The problem is that:
1. There is a lot with varying degrees of relevance.
2. Not all of the ClickHouse SQL functions/helpers are implemented in HogQL.